st2common/st2common/openapi.yaml

```diff
@@ -4002,6 +4075,16 @@ definitions:
     additionalProperties: False
   ExecutionFilters:
     type: object
+  Inquiry:
+    type: object
```

Comment:
Ack that, will follow your advice from https://github.com/StackStorm/st2/pull/3537 and add more detail here as well.